### PR TITLE
Convert feature-38 to new cnto additions format

### PR DESCRIPTION
**Pull request by Seb105**
https://github.com/CntoDev/cnto-additions-old/issues/38
~~NOT TESTED AS I DUNNO HOW TO BUILD NEW CNTO_ADDITIONS!~~

Tested, working with previous mentioned issues resolved.

### CODE REVIEW

**Comment by freghar** [comment]
> NOT TESTED AS I DUNNO HOW TO BUILD NEW CNTO_ADDITIONS!

Maybe https://github.com/CntoDev/cnto-additions/blob/dev/README.md ?

File: addons/littlebird_hud/fn_init.sqf [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	false   /* needRestart */
+] call CBA_settings_fnc_init;
+
+0 spawn {
+	if (!hasInterface) exitWith {};
+	waitUntil { !isNull player };
+	["vehicle", {
+		params ["_unit", "_newVehicle", "_oldVehicle"];
+		if (_newVehicle isKindOf "RHS_MELB_AH6M" OR _newVehicle isKindOf "Heli_Light_01_armed_base_F") then {
+			_newVehicle call a3aa_littlebird_hud_fnc_hud;
```
Comment (freghar) [inline addons/littlebird_hud/fn_init.sqf]: I'm assuming this won't work.

**Comment by Seb105** [comment]
> 
> 
> > NOT TESTED AS I DUNNO HOW TO BUILD NEW CNTO_ADDITIONS!
> 
> Maybe https://github.com/CntoDev/cnto-additions/blob/dev/README.md ?

Tested, it works.

**Comment by freghar** [comment]
Looks good, maybe aside from line endings in `fn_hud.sqf` and `config.cpp`. There are some possible optimizations that could be applied, but the current code looks good enough to me.

**Pull request merged**